validator/validator-main.protoascii

```diff
@@ -867,6 +867,25 @@ tags: {
     dispatch_key: NAME_VALUE_DISPATCH
   }
 }
+# AMP metadata, name=amp-cta-landing-page-type
+# Specifies the Single Page Story Ad landing page type
+tags: {
+  html_format: AMP4ADS
+  tag_name: "META"
+  spec_name: "meta name=amp-cta-landing-page-type"
+  mandatory_parent: "HEAD"
+  unique: true
+  attrs: {
+    name: "content"
+    mandatory_oneof: ['AMP', 'NOAMP', 'STORY']
```

Comment:
I think this needs to be in double-quotes, per the other examples in this file.

Reply:
Done.